CalibTracker/SiStripESProducers/plugins/services/modules.cc

```diff
@@ -40,3 +40,7 @@ DEFINE_FWK_SERVICE(SiStripBaseDelayGenerator);
 #include "FWCore/ServiceRegistry/interface/ServiceMaker.h"
 #include "CalibTracker/SiStripESProducers/interface/SiStripConfObjectGenerator.h"
 DEFINE_FWK_SERVICE(SiStripConfObjectGenerator);
+
+#include "FWCore/ServiceRegistry/interface/ServiceMaker.h"
+#include "CalibTracker/SiStripESProducers/interface/SiStripBadModuleFedErrService.h"
+DEFINE_FWK_SERVICE(SiStripBadModuleFedErrService);
```

Comment:
@davidlange6 can you clarify what is the correct approach? 
I think @suchandradutta reproduced the pattern of other dependent records in the same package where other
dependent ESSources are made (`SiStripNoisesFakeESSource`, `SiStripBadModuleConfigurableFakeESSource`,`SiStripLorentzAngleFakeESSource` etc)
using services like (`SiStripNoisesGenerator`, `SiStripBadModuleGenerator`, `SiStripLorentzAngleGenerator`).
thanks,

Marco